application/console/controllers/CronController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace console\controllers;
+
+use common\models\EmailQueue;
+use yii\console\Controller;
+
+class CronController extends Controller
+{
+    /**
+     * Retry sending emails from queue in small batch sizes
+     */
+    public function actionSendQueuedEmail()
+    {
+        try {
+            echo 'starting cron/send-queued-email' . PHP_EOL;
+
+            $batchSize = \Yii::$app->params['emailQueueBatchSize'];
+            $queued = EmailQueue::find()->orderBy(['last_attempt' => SORT_ASC])->limit($batchSize)->all();
+
+            if ($queued === null) {
```

Comment:
I think `$queued` will be an array, so you'll probably want to check for it to be an empty array rather than be null.